packages/components/src/components/notification-banner/notification-banner.css

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license
+ * Scale https://github.com/telekom/scale
+ *
+ * Copyright (c) 2021 Egor Kirpichev and contributors, Deutsche Telekom AG
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+:host {
+  --width: 100%;
+  --radius: var(--scl-radius-8);
+
+  /* Token not available, up for discussion */
+  --background-error: hsla(353.79999999999995, 100%, 88.6%, 0.35);
+  --background-warning: hsla(33.5, 100%, 75.1%, 0.3);
+  --background-informational: hsla(190.20000000000005, 100%, 92%, 0.75);
+  --background-success: hsl(98.80000000000001, 49.7%, 61%, 0.2);
+}
+
+.notification-banner {
+  border-radius: var(--radius);
+  width: var(--width);
+  position: relative;
+  box-shadow: 0px 0px 10px #e2e2e2;
+}
+
+.notification-banner--variant-error {
+  background-color: var(--background-error);
+}
+.notification-banner--variant-warning {
+  background-color: var(--background-warning);
+}
+.notification-banner--variant-informational {
+  background-color: var(--background-informational);
+}
+.notification-banner--variant-success {
+  background-color: var(--background-success);
+}
+
+.notification-banner__heading {
+  margin: 14px 48px 14px 48px;
+  font-weight: var(--scl-font-weight-bold);
+  line-height: 48px;
+}
+
+.notification-banner__text ::slotted(*) {
+  margin: -25px 0px -25px 0px;
+  padding: 0;
+}
+
+.notification-banner--has-text.notification-banner {
+  margin-top: -25px;
+}
+
+.notification-banner--has-no-text .notification-banner__link ::slotted(*) {
```

Comment:
this is not used, right?

Reply:
you resolved this question without answering - no problem, but i have to ask again: notification-banner--has-no-text is nowhere to be found but here. can you remove this then?